source/Plugins/ExpressionParser/Swift/SwiftUserExpression.cpp

```diff
@@ -320,10 +326,16 @@ void SwiftUserExpression::ScanContext(ExecutionContext &exe_ctx, Status &err) {
               log->Printf("    [SUE::SC] Argument name: %s",
                           template_arg_name.AsCString());
 
-            CompilerType concrete_type =
-                exe_ctx.GetProcessRef()
-                    .GetSwiftLanguageRuntime()
-                    ->GetConcreteType(frame, template_arg_name);
+            CompilerType concrete_type = template_arg_type;
```

Comment:
This and the snippet above (line 295) are exactly the same. Can we factor the logic out in a common function?